ecs-cli/modules/config/readwriter_v1.go

```diff
@@ -90,21 +116,19 @@ func readClusterConfig(path string, clusterConfigKey string, cliConfig *CLIConfi
 	cliConfig.Region = cluster.Region
 	cliConfig.Cluster = cluster.Cluster
 	cliConfig.ComposeServiceNamePrefix = cluster.ComposeServiceNamePrefix
+	// set prefixes to empty
+	cliConfig.CFNStackNamePrefix = ""
+	cliConfig.ComposeProjectNamePrefix = ""
```

Comment:
That's complicated. It has to do with the fact that the readINI function is called right before this function. So the ini function will fail to read the file, but unfortunately the old ini library sucks and it does not throw an error. So it reads the config file as empty- which causes the readINI function to set the prefixes to their defaults (which was the old behavior). So the readClusterConfig function needs to set the prefixes to empty.